packages/video_player/video_player/pigeons/messages.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:pigeon/pigeon_lib.dart';
+
+class VoidMessage {}
+
+class TextureMessage {
+  int textureId;
+}
+
+class LoopingMessage {
+  int textureId;
+  bool isLooping;
+}
+
+class VolumeMessage {
+  int textureId;
+  double volume;
+}
+
+class PositionMessage {
+  int textureId;
+  int position;
+}
+
+class CreateMessage {
+  String asset;
+  String uri;
+  String package;
+  String formatHint;
+}
+
+@HostApi()
+abstract class VideoPlayerApi {
+  VoidMessage initialize(VoidMessage msg);
+  TextureMessage create(CreateMessage msg);
+  VoidMessage dispose(TextureMessage msg);
+  VoidMessage setLooping(LoopingMessage msg);
+  VoidMessage setVolume(VolumeMessage msg);
+  VoidMessage play(TextureMessage msg);
+  PositionMessage position(TextureMessage msg);
+  VoidMessage seekTo(PositionMessage msg);
+  VoidMessage pause(TextureMessage msg);
+}
+
+void setupPigeon(PigeonOptions opts) {
```

Comment:
So in this whole file, only the import line and this function are required, everything else is freeform right? I wish we can somehow make this distinction clearer. Maybe we need a pub run to generate the dart template and this is pre-filled with a big dartdoc saying you need to fill this.

Reply:
This function isn't required.  All of these flags can be supplied at command-line.  I added this function so people can save arguments.

Reply:
right, understood. 

1- I just meant how can we provide a clearer affordance to this thing? Maybe just a doc in the readme is ok. But since we're telling people to create this file from scratch, it would be good to have some template available. 

2- kinda unrelated, can we tweak the name to not have `setup` in there? The generated java/objc code has the word `setup` in the static functions too and they mean pretty different things than this `setup`. Could be confusing if people cognitively mixed those things. Maybe just configurePigeon or some such.